presto/scripts/analyze_tables.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+print_help() {
+  cat << EOF
+
+Usage: $0 [OPTIONS]
+
+This script runs ANALYZE TABLE on all tables in a given schema to collect 
+statistics for query optimization. This helps Presto create better query plans 
+with lower peak memory usage.
+
+NOTE: This script assumes a Presto server is already running and the schema is
+pre-created.
+
+IMPORTANT: Currently, you should run this with CPU Presto (start_java_presto.sh 
+or start_native_cpu_presto.sh) rather than GPU Presto, as ANALYZE TABLE may not 
+be fully supported on GPU Presto.
+
+OPTIONS:
+    -h, --help              Show this help message.
+    -s, --schema-name       Name of the schema containing the tables to analyze (required).
+    -v, --verbose           Enable verbose output.
+    --host                  Presto coordinator hostname (default: localhost).
+    --port                  Presto coordinator port (default: 8080).
+    --user                  Presto user (default: test_user).
```

Comment:
Nit: Update options to match https://github.com/rapidsai/velox-testing/blob/ba7cc2b4fab35a2741977d4f932dd3fa0870c62a/presto/scripts/run_integ_test.sh#L32-L34